### PR TITLE
New Blog for Release 1.16 

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

docs update

* **What is the new behavior (if this is a feature change)?**

New Blog for Release 1.16

### CODE REVIEW

**Comment by Shelley-BaoYue** [comment]
cc @fisherxu @WillardHu @luomengY @zhiyingfang2022 @ZhengXinwei-F @wbc6080  
https://deploy-preview-517--kubeedge.netlify.app/blog/release-v1.16/

File: blog/release-v1.16/index.mdx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    Update the way mapper-framework generates mapper projects. The current execution script will only generate user-level code through dependent references. When the mapper project needs to be upgraded, it can be directly made by changing the version of mapper-framework package.
+
+Refer to the link for more details.([#5308](https://github.com/kubeedge/kubeedge/pull/5308), [#5326](https://github.com/kubeedge/kubeedge/pull/5326))
+
+### Integrate Redis and TDengine Database in DMI Data Plane
+
+Integrate redis and tdengine database in DMI data plane. The mapper project generated by mapper-framework has build-in ability to push data to redis and tdengine database. Users can push data directly through configuring device instance files.
+
+Database Field Definition:
+```json
```
Comment (WillardHu) [inline blog/release-v1.16/index.mdx:160]: The language can be specified as golang.  ```golang

File: blog/release-v1.16/index.mdx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+	// Poolsize of Redis database
+	// +optional
+	Poolsize int `json:"poo lsize,omitempty"`
+	// MinIdleConns of Redis database
+	// +optional
+	MinIdleConns int `json:"minIdleConns,omitempty"`
+}
+```
+
+```json
```
Comment (WillardHu) [inline blog/release-v1.16/index.mdx:182]: The language can be specified as golang.  ```golang

File: i18n/zh/docusaurus-plugin-content-blog/release-v1.16/index.mdx [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+	// Poolsize of Redis database
+	// +optional
+	Poolsize int `json:"poolsize,omitempty"`
+	// MinIdleConns of Redis database
+	// +optional
+	MinIdleConns int `json:"minIdleConns,omitempty"`
+}
+```
+
+```json
```
Comment (WillardHu) [inline i18n/zh/docusaurus-plugin-content-blog/release-v1.16/index.mdx:215]: The language can be specified as golang.  ```golang

File: i18n/zh/docusaurus-plugin-content-blog/release-v1.16/index.mdx [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+
+https://github.com/kubeedge/kubeedge/pull/5064
+
+### 基于Mapper-Framework的USB-Camera Mapper实现
+
+基于KubeEdge的Mapper-Framework，新版本提供了USB-Camera的Mapper样例，该Mapper根据USB协议的Camera开发，用户可根据该样例和Mapper-Framework更轻松地开发具体业务相关的Mapper。
+
+在样例中提供了helm chart包，用户可以通过修改usbmapper-chart/values.yaml部署UBS-Camera Mapper，主要添加USB-Camera的设备文件, nodeName, USB-Camera的副本数，其余配置修改可根据具体情况而定，通过样例目录中的Dockerfile制作Mapper镜像。
+
+```json
```
Comment (WillardHu) [inline i18n/zh/docusaurus-plugin-content-blog/release-v1.16/index.mdx:241]: The language can be specified as yaml.  ```yaml

Comment (Shelley-BaoYue) [reply i18n/zh/docusaurus-plugin-content-blog/release-v1.16/index.mdx]: Done.

File: blog/release-v1.16/index.mdx (hunk 1 above)
Comment (Shelley-BaoYue) [reply blog/release-v1.16/index.mdx:160]: I compared json with golang. I think `json` is more suitable both in display and its meaning here.

image

Comment (WillardHu) [reply blog/release-v1.16/index.mdx:160]: Okey

**Review by fisherxu — COMMENTED** [review]
/approve

Thanks for the blog.

**Pull request merged**